### ISSUE TITLE
Add soundhw 

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#125]
Originally reported of Google Code with ID 69.

```
Hi all!

I'm starting to use the ganeti as a tool for managing my virtual machines.
I can appreciate the power and simplicity that this tool has.
First of all, congratulations to all who contributed and continue
contributing to the growth of this project.
I saw two needs that could come as improvement. These are the parameters of
qemu-dm: -soundhw and -std-vga.
It would be interesting to be able to set these parameters in instances
through the ganeti.

Thanks in advance,


Renato Fernandes
WEBTraffic
```
Originally added on 2009-08-31 19:06:49 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2009-10-23 12:54:11 +0000 UTC.

Added Labels: Type-Enhancement Priority-Low
Added to Milestone: Release2.2

**Comment by Ganeti-Issues-Migrator** [comment]
```
This unfortunately didn't make the 2.2 release… Targetting to 2.3.
```
Originally added on 2010-07-29 20:48:24 +0000 UTC.

Changed State: Accepted
Added to Milestone: Release2.3
Removed from Milestone: Release2.2

**Comment by Ganeti-Issues-Migrator** [comment]
```
De-targetting these issues, were not implemented in 2.3.
```
Originally added on 2011-02-18 13:43:21 +0000 UTC.

Added to Milestone: Unplanned
Removed from Milestone: Release2.3

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2011-08-05 14:45:42 +0000 UTC.

Removed from Milestone: Unplanned

**Comment by Ganeti-Issues-Migrator** [comment]
```
I am also waiting on this issue, but the format has changed from -vga-std , to -vga (std|cirrus|vmware) in de KVM hypervisor.

Windows guests do not work correctly without this option , which makes ganeti a whole less useful for me. Especially RDP is hogging bandwidth because of the display driver problem.

```
Originally added on 2012-03-11 12:33:21 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2012-10-22 08:57:07 +0000 UTC.

Changed State: Released

**Comment by Ganeti-Issues-Migrator** [comment]
```
Undo wrong status update to 'released'.
```
Originally added on 2012-10-22 09:01:59 +0000 UTC.

Changed State: Accepted

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2013-01-11 18:05:22 +0000 UTC.

Added to Milestone: Release2.7

**Comment by Ganeti-Issues-Migrator** [comment]
```
This is fixed in 2.7.0~beta2
```
Originally added on 2013-04-08 14:05:59 +0000 UTC.

Changed State: Fixed

**Comment by Ganeti-Issues-Migrator** [comment]
```
While this was fixed for "kvm", it wasn't for xen-hvm. Thus retargeting to 2.8 for Xen-HVM while the kvm fix is part of 2.7.
```
Originally added on 2013-04-08 20:13:35 +0000 UTC.

Changed State: Accepted
Added to Milestone: Release2.8
Removed from Milestone: Release2.7

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2013-04-10 18:09:23 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2013-05-08 08:24:46 +0000 UTC.

Added Labels: SmallTask

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2013-08-20 08:26:12 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2013-08-20 08:28:50 +0000 UTC.

Added to Milestone: Release2.9
Removed from Milestone: Release2.8

**Comment by Ganeti-Issues-Migrator** [comment]
```
Soundhw support has been written (commit b5cad58) and should be released in Ganeti 2.10.
```
Originally added on 2013-10-19 08:50:19 +0000 UTC.

Changed State: Fixed
Added to Milestone: Release2.10
Removed from Milestone: Release2.9

**Comment by Ganeti-Issues-Migrator** [comment]
```
2.10.0 with fixes for these bugs has just been released.
```
Originally added on 2014-02-20 13:25:23 +0000 UTC.

Changed State: Released
